App.js

```diff
@@ -4,9 +4,18 @@ import { Asset } from 'expo-asset';
 import * as Font from 'expo-font';
 import React, { useState } from 'react';
 import { Platform, StatusBar, StyleSheet } from 'react-native';
+import * as Sentry from 'sentry-expo';
 import AppNavigator from './navigation/AppNavigator';
 import { Container } from './styled/shared';
 
+Sentry.init({
+  dsn: 'https://9b39ef5b02394775a6e939254ed55a0e@o306199.ingest.sentry.io/4525076',
+  enableInExpoDevelopment: true,
```

Comment:
Change this to `false` by default so we don't use up too many of BP's events, make release `v1.1.1`, and environment switch on env with `process.env.NODE_ENV`